netdata-installer.sh

```diff
@@ -41,6 +41,45 @@ fi
 # make sure /etc/profile does not change our current directory
 cd "${NETDATA_SOURCE_DIR}" || exit 1
 
+# -----------------------------------------------------------------------------
+# figure out an appropriate temporary directory
+_cannot_use_tmpdir() {
+  local testfile ret
+  testfile="$(TMPDIR="${1}" mktemp -q -t netdata-test.XXXXXXXXXX)"
+  ret=0
+
+  if [ -z "${testfile}" ] ; then
+    return "${ret}"
+  fi
+
+  if /bin/echo -e "#!/bin/sh\necho SUCCESS\n" > "${testfile}" ; then
```

Comment:
This doesn't work if it is executed in bash as it tried to evaluate the `"!"` as a history event. It will work if you use `'` instead to make it literal.